main/src/main/java/com/google/android/apps/muzei/util/SvgPathParser.java

```diff
@@ -149,6 +149,21 @@ public Path parsePath(String s) throws ParseException {
                     mCurrentPoint.set(tempPoint1);
                     break;
                 }
+                
```

Comment:
Ah, no the extra line is fine - it is the spaces on that extra line that are different from the rest of the file.